app/src/main/java/org/hisp/dhis/android/dataentry/form/section/viewmodels/FormItemViewModelFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.hisp.dhis.android.dataentry.form.section.viewmodels;
+
+import android.database.Cursor;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import org.hisp.dhis.android.core.common.ValueType;
+
+public interface FormItemViewModelFactory {
```

Comment:
We might try to go further, but I don't think there is any reason to wait until post MVP. It might be too expensive to change after release, because models are quite fundamental to the data entry.